src/service/standard-actions-impl.js

```diff
@@ -205,6 +216,92 @@ export class StandardActions {
     throw user().createError('Unknown AMP action ', method);
   }
 
+  /**
+   * Handles the copy to clipboard action
+   * @param {!./action-impl.ActionInvocation} invocation
+   * @return {!null}
```

Comment:
We don't really need this to have a return type, right?

Reply:
Also the `!` prefix doesn't do anything/should be left out now that we've switched off of CC.

Reply:
Yah, we can just remove this and the return statement.

Reply:
Sure, I will correct it.